.github/workflows/check-maintainers-sorted.yaml

```
@@ -0,0 +1,31 @@
+name: "Check that maintainer list is sorted"
+
+on:
+  pull_request_target:
+    branches:
+      - master
+    paths:
+      - 'maintainers/maintainer-list.nix'
```

Nits: For consistency with the YAML format I think we should use double quotes.

```suggestion
      - "maintainers/maintainer-list.nix"
```

---

No problem, just ignore my comments